billy-andorra/src/main/java/com/premiumminds/billy/andorra/services/builders/impl/ADSimpleInvoiceBuilderImpl.java

```diff
@@ -69,11 +68,11 @@ protected void validateInstance() throws BillyValidationException {
                                  ADGenericInvoiceBuilderImpl.LOCALIZER.getString("field.clientType"));
         super.validateInstance();
 
-        if (i.getClientType() == CLIENTTYPE.CUSTOMER && i.getAmountWithTax().compareTo(new BigDecimal(1000)) >= 0) {
-            throw new BillySimpleInvoiceException("Amount > 1000 for customer simple invoice. Issue invoice");
+        if (i.getClientType() == CLIENTTYPE.CUSTOMER && i.getAmountWithTax().compareTo(new BigDecimal("40000")) >= 0) {
```

Comment:
shoud these hardcoded values be moved to configuration options ?

Reply:
That's a good idea :+1: